File: backend/Models/Receita.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Models;

public class Receita
{
	public int Id { get; set; }

    [Required]
    [MaxLength(150)]
	public required string Tipo { get; set; }

    [Required]
    [MaxLength(150)]
	public required string Origem { get; set; }

	public DateTime DataEntrada { get; set; }

	public decimal Valor { get; set; }

	// Relacionamento com Projeto
	public int ProjetoId { get; set; }

	public Projeto? Projeto { get; set; }
}

File: android/app/src/main/java/com/example/radarfinanceiro/models/Receita.java
package com.example.radarfinanceiro.models;

public class Receita {

    private int id;
    private String tipo;
    private String origem;
    private String dataEntrada;
    private double valor;
    private int projetoId;

    public int getId() {
        return id;
    }

    public String getTipo() {
        return tipo;
    }

    public String getOrigem() {
        return origem;
    }

    public String getDataEntrada() {
        return dataEntrada;
    }

    public double getValor() {
        return valor;
    }

    public int getProjetoId() {
        return projetoId;
    }
}
